ONSdigital/eq-survey-runner
# Response Guidance

This PR adds basic guidance to responses.
### Changes
- added guidance data to schema for various questions
- added section styling
- added guidance styling
- added guidance expand interaction
- re-worked typed inputs (eg. currency, percentage)
- moved rendering if `input`, `label` and `textarea` into macros
### Testing
- `npm run compile`
- Visit `/questionnaire/mci/`
- Check guidance works by clicking "show guidance for this question"
